### PR TITLE
add version to scripts

### PR DESCRIPTION
**Pull request by henrykironde**
In this PR we are adding  script version (1.0) and retriever minimum version (2.0) attributes,
Change `version.py` to look for the version from the script and print it in `version.txt`

### CODE REVIEW

File: version.txt [hunk 1]
```diff
@@ -1,81 +1,79 @@
 v1.9.0-dev
-AvianBodySize.json
-FishParasiteHosts.json
-Steppe_plants_2013.json
-CRC_avianbodymass.py
```
Comment (henrykironde) [inline version.txt]: CRC_avianbodymass.py I guess we do not need this in version.txt

File: version.txt [hunk 2]
```diff
@@ -1,81 +1,79 @@
...
-DelMoral2010.json
-npn.py
-Matter2014.json
-bioclim_2pt5.py
-Palmer2007.json
-Adler2007.json
-Ramesh2010.json
-MammalMR2010.json
-PlantTaxonomy.json
-MammalDIET.json
```
Comment (henrykironde) [inline version.txt]: @goelakash, Line [43 MammalDIET.json](https://github.com/weecology/retriever/pull/635/files#diff-5fb01c6f8b54d96c62af3415c51ffad3L43) and line [54 MammalDiet.script](https://github.com/weecology/retriever/pull/635/files#diff-5fb01c6f8b54d96c62af3415c51ffad3L54) should these be having the same name ? Wondering if Capitals make a difference

File: version.txt (hunk 1 above)
Comment (ethanwhite) [inline version.txt]: Yeah, I think we can go ahead and just remove this script entirely.

**Comment by ethanwhite** [comment]
LGTM.

@goelakash - take a look and if it looks to you why don't you go ahead and do your first official merge.

**Comment by henrykironde** [comment]
@goelakash could we get this in so that I get to test the next PR

**Comment by ethanwhite** [comment]
I'm happy to merge this if it's holding things up.

**Comment by henrykironde** [comment]
yes, go ahead and merge this. The other PR is no urgent at all

**Pull request merged**